### PR TITLE
Update cms-learning-resources.xml

### PR DESCRIPTION
**Pull request by pherterich**
addressing #756 on -dev first for approval/review by @katilp

### CODE REVIEW

**Comment by katilp** [comment]
I can't see this on dev portal in learning resources: neither in
http://opendatadev.cern.ch/search#%5B%5B%22%2B%22%2C%22collection%22%2C%22CMS-Learning-Resources%22%5D%2C%5B%22!%22%2C%22collection%22%2C%22CMS-Learning-Resources%22%5D%5D
nor in http://opendatadev.cern.ch/collection/CMS-Learning-Resources
Am I looking in the wrong place?

**Comment by pherterich** [comment]
@tiborsimko hasn't included my changes yet, but that's where it's supposed to show up then

**Comment by tiborsimko** [comment]
@katilp This pull request has not been merged yet, hence it is not deployed on DEV yet either.  

If you'd like to check the record text beforehand, please do it here:

- https://github.com/cernopendata/opendata.cern.ch/pull/760/files

(You could even commment there on some particular points, if something needs fixing...)

File: invenio_opendata/testsuite/data/cms/cms-learning-resources.xml [hunk 1]
```diff
@@ -228,4 +228,47 @@
...
+    </datafield>
+    <datafield tag="260" ind1=" " ind2=" ">
+      <subfield code="b">University of Helsinki</subfield>
+      <subfield code="c">2014</subfield>
+    </datafield>
+    <datafield tag="520" ind1=" " ind2=" ">
+      <subfield code="a">This introductory course by S. Lehti is aimed for students with no prior knowledge of computing methods in high-energy physics. Prior knowledge of some programming language is needed. The course goes through basics of programming languages in high energy physics (FORTRAN, C++, python), software for analysis and visualisation (ROOT), some software for calculating cross sections and branching ratios, simulation of collisions (event generation), basics of CMS software (CMSSW) and an introduction to grid computing. This course is developed in the University of Helsinki for students interested in experimental particle physics.</subfield>
+    </datafield>
+    <datafield tag="516" ind1=" " ind2=" ">
+      <subfield code="a">All exercises can be done with the CMS open data image available from the portal.</subfield>
```
Comment (katilp) [inline invenio_opendata/testsuite/data/cms/cms-learning-resources.xml]: We have not tested but we think that they can be done with the image. So this needs to be rephrased... maybe "All exercises are compatible with CMS open data image, but have not yet been tested with it"

File: invenio_opendata/testsuite/data/cms/cms-learning-resources.xml [hunk 2]
```diff
@@ -228,4 +228,47 @@
...
+    </datafield>
+    <datafield tag="556" ind1=" " ind2=" ">
+      <subfield code="a">Includes relevant warnings for the Univ. Helsinki specific part of the instructions (e.g. use of local computing resources).</subfield>
+    </datafield>
```
Comment (katilp) [inline invenio_opendata/testsuite/data/cms/cms-learning-resources.xml]: This statement is probably not necessary.

File: invenio_opendata/testsuite/data/cms/cms-learning-resources.xml [hunk 3]
```diff
@@ -228,4 +228,47 @@
...
+    <datafield tag="856" ind1="4" ind2=" ">
+      <subfield code="u">http://www.helsinki.fi/~slehti/ComputingMethodsInHEP/ComputingMethodsInHEP.html</subfield>
+      <subfield code="y">Go to the Computing Methods in High Energy Physics course</subfield>
+    </datafield>
+    <datafield tag="980" ind1=" " ind2=" ">
+      <subfield code="a">CMS-Learning-Resources</subfield>
+    </datafield>
+    <datafield tag="980" ind1=" " ind2=" ">
+      <subfield code="b">Education</subfield>
+    </datafield>
```
Comment (katilp) [inline invenio_opendata/testsuite/data/cms/cms-learning-resources.xml]: Should this be research? This course is certainly useful for anyone for plan to start research. It is education for research....

File: invenio_opendata/testsuite/data/cms/cms-learning-resources.xml [hunk 4]
```diff
@@ -228,4 +228,47 @@
...
+    <datafield tag="245" ind1=" " ind2=" ">
+      <subfield code="a">Computing Methods in High Energy Physics</subfield>
+    </datafield>
+    <datafield tag="260" ind1=" " ind2=" ">
+      <subfield code="b">University of Helsinki</subfield>
+      <subfield code="c">2014</subfield>
+    </datafield>
+    <datafield tag="520" ind1=" " ind2=" ">
+      <subfield code="a">This introductory course by S. Lehti is aimed for students with no prior knowledge of computing methods in high-energy physics. Prior knowledge of some programming language is needed. The course goes through basics of programming languages in high energy physics (FORTRAN, C++, python), software for analysis and visualisation (ROOT), some software for calculating cross sections and branching ratios, simulation of collisions (event generation), basics of CMS software (CMSSW) and an introduction to grid computing. This course is developed in the University of Helsinki for students interested in experimental particle physics.</subfield>
+    </datafield>
```
Comment (katilp) [inline invenio_opendata/testsuite/data/cms/cms-learning-resources.xml]: high-energy vs high energy?

Comment (pherterich) [inline invenio_opendata/testsuite/data/cms/cms-learning-resources.xml]: Good point, in the title, I copied the original from the homepage and then I tried to follow @RaoOfPhysics wish for the rest.

File: invenio_opendata/testsuite/data/cms/cms-learning-resources.xml (hunk 3 above)
Comment (pherterich) [inline invenio_opendata/testsuite/data/cms/cms-learning-resources.xml]: I can make it both I think. @tiborsimko would that crash anything?

Comment (jirikuncar) [inline invenio_opendata/testsuite/data/cms/cms-learning-resources.xml]: @pherterich it should not crash anything.

**Comment by pherterich** [comment]
Included the comments from @katilp 
Might as well go straight into production now I think.

**Comment by tiborsimko** [comment]
Thanks. squashed, amended commit log message and merged as 614545d.

**Pull request closed without merging**